composer.json

```diff
@@ -60,31 +60,5 @@
         "typo3/cms-tstemplate": "dev-main",
         "typo3/cms-viewpage": "dev-main",
         "typo3/cms-workspaces": "dev-main"
-    },
-    "scripts": {
```

Comment:
Can't verify this right now, but how does the runTests work exactly, doesn't it maybe utilize these composer commands?

Reply:
There is a Makefile now.

Reply:
And in the GitHub action we call runTests.sh directly, not via Composer.

Reply:
And no, runTests.sh does not use our defined Composer scripts. It is independent from that.

Reply:
I reintroduced the composer commands. We can keep them for those who cannot or don't want to use make.

Reply:
We should not hold multiple definitions, those have to maintained on changes (which may be forgotten). Either call the Composer scripts in the Makefile or remove one of them (Composer or Makefile).

Reply:
Personally, I would stick to the Composer scripts here and would omit the Makefile. Composer scripts are well-known and work on every platform.

Reply:
composer has big disadvantages as you have to run them either on local php or in ddevs PHP. if you run them in ddevs PHP - as I do - you cannot address the runscripts.sh in there as it will also call its own docker container.

Make files are also very wide spread and they do not have the dependency on local PHP or ddev

Reply:
Also they have the very nice feature of automatic documentation of all available commands. And we already descided on them in render-guides. Whoever does not want to use make for whatever reason can copy the commands and call the runscripts directly

Reply:
So I removed the composer scripts once more. I had understood that you wanted them back

Reply:
So, the recommended way would be to run the scripts with `ddev exec make <command>`. Then we also need the
'build-essential` package added to `webimage_extra_packages` in ddev config which provides `make`.

This should then also be reflected somewhere in the README or similar.

Reply:
no you have to do `make command`

Otherwise you would call docker in docker

Reply:
So @brotkrueml would you like to switch to the guides without modern testing?

Reply:
Ok, fine, removed the testing again

Reply:
Don't understand? What has the execution of commands with Composer or make to do with modern testing? It does not matter if done via Composer or make?

Reply:
But if you think this is the way, then do it.

Reply:
So I removed the testing for now an put it in its own PR. Can we get this one merged?